src/WormholeGUID.sol

```diff
@@ -20,13 +20,21 @@ pragma solidity 0.8.9;
 struct WormholeGUID {
     bytes32 sourceDomain;
     bytes32 targetDomain;
-    address receiver;
-    address operator;
+    bytes32 receiver;
+    bytes32 operator;
     uint128 amount;
     uint80 nonce;
     uint48 timestamp;
 }
 
+function bytes32ToAddress(bytes32 addr) pure returns (address) {
+    return address(uint160(uint256(addr)));
+}
+
+function addressToBytes(address addr) pure returns (bytes32){
```

Comment:
Some nits:
- should be `internal`
- should be named `addressToBytes32` for consistency
- needs a space between `returns (bytes32)` and `{` for consistency

Reply:
Ah yes, nevermind. We could just add a `// solhint-disable-next-line func-visibility` on top of it to mute solhint warning.